text/0096-contract-metadata.md

```diff
@@ -0,0 +1,213 @@
+# Casper On-Chain Contract Metadata Standard
```

Comment:
@mpapierski, Michael is asking if this CEP is in conflict with the contract metadata feature in the VM2. Could you please comment on that.

Reply:
@mpapierski can we put the native contract metadata under the same CEP? Basically it could follow the same standard.